docs/getting-started/v3-migration.md

```diff
@@ -98,36 +98,41 @@ Chart.js 3.0 introduces a number of breaking changes. Chart.js 2.0 was released
 
 ### Renamed
 
-* `helpers.clear` was renamed to `helpers.canvas.clear`
-* `helpers.drawRoundedRectangle` was renamed to `helpers.canvas.roundedRect`
-* `helpers.callCallback` was renamed to `helpers.callback`
-* `helpers.getValueOrDefault` was renamed to `helpers.valueOrDefault`
-* `helpers.getValueAtIndexOrDefault` was renamed to `helpers.valueAtIndexOrDefault`
-* `helpers.easingEffects` was renamed to `helpers.easing.effects`
-* `helpers.log10` was renamed to `helpers.math.log10`
+* `Chart.Animation.animationObject` was renamed to `Chart.Animation`
```

Comment:
It would seem the most common way to do this is to ignore the case when alphabetizing

https://english.stackexchange.com/questions/314320/guide-to-alphabetizing-upper-case-versus-lower-case

If we're going to reorder we should probably change the other sections to match

Reply:
Done, I didn't even notice I used case sensitive sorter. Good catch :)